cli/src/main/resources/help/command.txt

```diff
@@ -46,7 +46,13 @@ DESCRIPTION
 
   data-source --jndi-name=my-new-ds --min-pool-size=11 --max-pool-size=22
 
-  Tab-completion will help completing operation and property names.
+  
+ARGUMENTS
```

Comment:
Seems to me that the ARGUMENTS should contain what the "Managing Operations" contains. This document is the documentation of "command" not of the generated command.
Then the Managing Operation should be named "Generated command"